2020/hamming-codes/spanish/sentence_translations.json

```
@@ -479,7 +479,7 @@
  {
   "input": "And actually, we should be clear, if the receiver sees an odd parity, it doesn't necessarily mean there was just one error, there might have been 3 errors, or 5, or any other odd number, but they can know for sure that it wasn't 0.",
   "model": "nmt",
-  "translatedText": "Y, de hecho, debemos ser claros: si el receptor ve una paridad impar, no significa necesariamente que hubo solo un error, puede haber habido 3 errores, o 5, o cualquier otro número impar, pero pueden estar seguros. que no era 0.",
+  "translatedText": "Y, de hecho, debemos ser claros, si el receptor ve una paridad impar, no significa necesariamente que hubo solo un error, puede haber habido 3 errores, o 5, o cualquier otro número impar, pero se puede estar seguro de que no era 0.",
```

```suggestion
  "translatedText": "Y, de hecho, debemos ser claros, si el receptor ve una paridad impar, no significa necesariamente que hubo solo un error, puede haber habido 3 errores, o 5, o cualquier otro número impar, pero puede estar seguro de que no era 0.",
```
  I believe he's talking about the `receptor` which would mean that the `se` is unnecessary.